client/styles/.gitignore

```diff
@@ -0,0 +1 @@
+storybook.css
```

Comment:
Love this! ❤️  We definitely don't want to include temporary/autogenerated files in the repo.